moment-recur.js

```diff
@@ -590,6 +601,30 @@
             }
         };
 
+        // Checks for rule (by passing measure) and exception (by passing date)
+        Recur.prototype.has = function(dateOrRule) {
```

Comment:
And yes, I realize this is rather in keeping with the way forget works.  But I am not really a fan of the way that function overloads a string parameter to be either a parseable date or a measure.  My concern here is that someone will use 'has' with a mal formatted date, and this function will rather blindly ignore that programmer error and try and match it against a measure, which isn't right either.